### ISSUE TITLE
aesgcm URL not decoded

### ISSUE THREAD

**Issue by schnedann** [psi-plus/main#797]
When I send a picture to another OMEMO user (using Psi+), it works fine (other Client is Android Conversations), When that user sends a picture to me, I just get an AES encoded URL String (aesgcm://xyz...) which I can not use any further. So it seems to me like a bug in PSI, not decoding the aes encoded URL.

Can you verify this as a bug?
Me using Psi+ 1.5.1477

**Comment by Vitozz** [comment]
Unfortunately this is not a bug. Omemo file transfer mechanism is not yet implemented.

**Comment by schnedann** [comment]
Ok, then, it's a feature request now :-) [as Psi+ is the only client I know which I can talk with OTR and OMEMO users at the same time...]

**Comment by Neustradamus** [comment]
@schnedann: Have you tested with latest build?

**Comment by schnedann** [comment]
yes, in the meantime the links displayed by psi+ work. Of course its a pain to copy and paste em by hand to e.g. a browser to actually view the picture. Why not display images in PSI? When I send something its displayed... so it actually has the technical ability to do so.

**Comment by Neustradamus** [comment]
@schnedann: Yes...

The image is in Psi/Psi+ profile folder.

And it is linked to:
- https://github.com/psi-im/psi/issues/487
